install/context.go

```diff
@@ -0,0 +1,11 @@
+package install
+
+import "github.com/repometric/lhman/catalog"
+
+// Context describes all necessary parameters for installation
+type Context struct {
```

Comment:
I suggest improving the struct name, now it's too generic.

Reply:
Context struct is in `install` package, so this eliminates any problems with naming